src/pages/home/report/ReportActionItem.js

```diff
@@ -87,15 +87,24 @@ class ReportActionItem extends Component {
         this.textInput.focus();
     }
 
+    /**
+     * Check if the message is deleted
+     * @returns {boolean}
+     */
+    isDeleted() {
+        // A deleted comment has either an empty array or an object with html field with empty string as value
+        return lodashGet(this.props.action, 'message[0].html', '') === '';
+    }
```

Comment:
Move this to ReportUtils with `isDeletedAction`

Reply:
Done